pkgs/development/tools/functiontrace-server/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, rustPlatform, fetchCrate, stdenv, darwin }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "functiontrace-server";
+  version = "0.5.2";
+
+  src = fetchCrate {
+    inherit pname version;
+    hash = "sha256-p6ypMfg99ohQCyPB2O0wXbGmPvD2K9V3EnFDd5dC6js=";
+  };
+
+  cargoHash = "sha256-3tLjW7yiS1dNsV81KUZbfN2pvYT9kqiC62nWFid2NH8=";
+
+  buildInputs = lib.optionals stdenv.isDarwin
+    [ darwin.apple_sdk.frameworks.CoreFoundation ];
+
+  meta = with lib; {
+    description =
+      "The server component that FunctionTrace (functiontrace.com) clients will spawn and connect to";
```

Comment:
The description is too lengthy IMO (and with little information), what about: `FunctionTrace graphical python profiler server`